projects/sandbox/train/tests/test_validation.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+import torch
+from train import validation
+
+
+def test_background_recall():
+    metric = validation.BackgroundRecall(10, 10, 3)
+    background = torch.zeros((31,))
+
+    # after pooling, the top 3 values should come
+    # out to 2, 1.5, and 0.8
+    background[3] = 1
+    background[4] = 1.5
+    background[10] = 2
+    background[14] = 0.5
+    background[18] = 0.9
+    background[22] = 0.8
+    background[30] = 3  # this will get trimmed
```

Comment:
is this a `torch.nn.functional.maxpool1d` artifact?

Reply:
Yeah exactly, actually didn't know this until I built this but I guess it adopts the convention of sloughing data off the end that can't be fit into a full kernel rather than doing the op on a shorter kernel. Good to keep in mind for the future I guess